src/pages/guides/getting_started/changelog.md

```diff
@@ -22,6 +22,26 @@ contributors:
 
 # Changelog
 
+## 2025-11-24
+
+### Code Playground Updates
+
+The Adobe Express Code Playground has received a major update with the release of a new set of features and improvements, which this site now reflects.
+
+- The Code Playground now supports [Cloud-based session persistence](./code-playground-workflow.md#session-management) (instead of the previous local storage) allowing you to save your work and resume where you left off everywhere. The new **More** menu contains items to manage your sessions (rename or delete), or star new ones.
```

Comment:
Rephrase. I would like to emphasize on multiple sessions instead of cloud persistence.
 Perhaps 'Code Playground now supports multiple sessions backed up in the cloud, allowing you to save your work and resume where you left off anywhere. Check out the **More** menu to manage your sessions (rename or delete).

(FYI, we dont allow devs to star/bookmark sessions)